src/SearchField/SearchField.scss

```diff
@@ -1,83 +1,33 @@
-$search-field-clear-btn-width: 30px;
-$search-field-border-radius: 30px;
-
-.search-field {
+.pgn-searchfield {
```

Comment:
Let's use `.pgn__` as the prefix. See the docs for TransitionReplace here: https://github.com/edx/paragon/tree/32773a388ddfc74d422706273ec6b06e6721d408/src/TransitionReplace It looks like this isn't fully consistent, but I think using underscores is helpful to indicate that this is somewhat private. What do you think?

Reply:
Sounds good to me; will update.